### PR TITLE
add recommendation for Tensor Cores

### PR DESCRIPTION
**Pull request by gaoteng-git**
image

### CODE REVIEW

File: tb_plugin/torch_tb_profiler/profiler/module_parser.py [hunk 1]
```diff
@@ -156,3 +157,4 @@ def aggregate(self, tid2tree):
         self.op_list_groupby_name_input = list(agg_result[1].values())
         self.stack_lists_group_by_name = stack_lists_group_by_name
         self.stack_lists_group_by_name_input = stack_lists_group_by_name_input
+        self.tc_eligible_ops = sum(map(lambda op : op.tc_eligible, ops))
```
Comment (guotuofeng) [inline tb_plugin/torch_tb_profiler/profiler/module_parser.py:160]: seems list comprehension is faster than map if using lambda? https://stackoverflow.com/questions/1247486/list-comprehension-vs-map

Comment (gaoteng-git) [reply tb_plugin/torch_tb_profiler/profiler/module_parser.py]: @guotuofeng Cool! What about "sum([1 if op.tc_eligible else 0 for op in ops])"?

Comment (gaoteng-git) [reply tb_plugin/torch_tb_profiler/profiler/module_parser.py]: @guotuofeng Compared the speed, they are nearly same(<20% difference). I'll use "sum([1 if op.tc_eligible else 0 for op in ops])" for easy to understand.

File: tb_plugin/torch_tb_profiler/profiler/data.py [hunk 2]
```diff
@@ -229,6 +231,21 @@ def analyze(self):
         self._analyze_distributed_metrics()
         self._analyze_gpu_metrics()
 
+        # Tensor Cores feature is available on GPU cards with compute capability >= 7.0
+        # https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#features-and-technical-specifications
+        if self.device_props is not None and len(self.device_props) > 0:
```
Comment (guotuofeng) [inline tb_plugin/torch_tb_profiler/profiler/data.py:236]: if self.device_props is enough.

Comment (gaoteng-git) [reply tb_plugin/torch_tb_profiler/profiler/data.py]: Thanks! Done.

**Review by guotuofeng — APPROVED** [review]

**Pull request merged**